src/libraries/Silk.Net.Windowing/Silk.Net.Windowing.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>$(SilkTargetFramework)</TargetFrameworks>
+        <Nullable>enable</Nullable>
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <WarningsAsErrors>true</WarningsAsErrors>
+        <LangVersion>9.0</LangVersion>
+
+        <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+    </PropertyGroup>
```

Comment:
Todo: verify with rest of project setup

Reply:
This was based on silk.net.Maths

Reply:
Yeah, but the 3.0 setup is quite different. I'll review it when everything else is done, just left this here to remind myself

Reply:
I mean it was pulled from the Silk.net.Maths implementation currently sitting in the directory right beside it, I figured at worst it would not completely break anything